onnxruntime/core/providers/cpu/signal/utils.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/framework/tensor.h"
+
+namespace onnxruntime {
+namespace signal {
+
+template <typename T>
+static T get_scalar_value_from_tensor(const Tensor* tensor) {
```

Comment:
I'm wondering about how other operators do (Reduce function for example). This code probably already exists somewhere else.